app/src/main/java/org/thoughtcrime/securesms/database/model/NotificationMmsMessageRecord.java

```diff
@@ -64,50 +61,44 @@ public NotificationMmsMessageRecord(long id, Recipient conversationRecipient,
     this.transactionId   = transactionId;
   }
 
-  public byte[] getTransactionId() {
-    return transactionId;
-  }
-  public int getStatus() {
-    return this.status;
-  }
-  public byte[] getContentLocation() {
-    return contentLocation;
-  }
-  public long getMessageSize() {
-    return (messageSize + 1023) / 1024;
-  }
-  public long getExpiration() {
-    return expiry * 1000;
-  }
+  public byte[] getTransactionId() { return transactionId; }
+
+  public int getStatus() { return this.status; }
+
+  public byte[] getContentLocation() { return contentLocation; }
+
+  public long getMessageSize() { return (messageSize + 1023) / 1024; }
+
+  public long getExpiration() {  return expiry * 1000; }
 
   @Override
-  public boolean isOutgoing() {
-    return false;
-  }
+  public boolean isOutgoing() { return false; }
 
   @Override
-  public boolean isPending() {
-    return false;
-  }
+  public boolean isPending() { return false; }
 
   @Override
-  public boolean isMmsNotification() {
-    return true;
-  }
+  public boolean isMmsNotification() { return true; }
 
   @Override
-  public boolean isMediaPending() {
-    return true;
-  }
+  public boolean isMediaPending() { return true; }
 
   @Override
   public SpannableString getDisplayBody(@NonNull Context context) {
-    if (status == MmsDatabase.Status.DOWNLOAD_INITIALIZED) {
-      return emphasisAdded(context.getString(R.string.NotificationMmsMessageRecord_multimedia_message));
-    } else if (status == MmsDatabase.Status.DOWNLOAD_CONNECTING) {
-      return emphasisAdded(context.getString(R.string.NotificationMmsMessageRecord_downloading_mms_message));
-    } else {
-      return emphasisAdded(context.getString(R.string.NotificationMmsMessageRecord_error_downloading_mms_message));
-    }
+    // To the best of my knowledge and ability to trace through the code this never gets used in any
+    // way that is ever displayed to the user. Previously, it would return:
+    //
+    //     - NotificationMmsMessageRecord_multimedia_message ("Multimedia message")
+    //         - If `status` was DOWNLOAD_INITIALIZED,
+    //     - NotificationMmsMessageRecord_downloading_mms_message ("Downloading MMS message")
+    //         - if `status` was DOWNLOAD_CONNECTING, and
+    //     - NotificationMmsMessageRecord_error_downloading_mms_message ("Error downloading MMS message, tap to retry")
+    //         - otherwise.
+    //
+    // As i.) the user never sees these, ii.) the strings are flagged for removal in SS-40, and
+    // iii.) SES-562 will provide updated attachment download controls - I'm going to delete the
+    // above strings as part of SS-40 and return an empty SpannableString which is never seen or
+    // used just to satisfy the method contract.
+    return EMPTY_SS;
```

Comment:
Any reason to keep the method at all?
The  base class already satisfies the contract. Do we need to make sure this specific instance returns an empty string as opposed to removing the function entirely and leaving it to the base class, especially if this doesn't get called?

Reply:
No - I thought we had to satisfy the interface but it was an optional override (the base method falls back to `MessageRecord`). Method removed & all seems to be working just fine without it.